### PR TITLE
Add Iconfinder

### PR DESCRIPTION
**Pull request by Anshuman-Verma**
**http://www.iconfinder.com/**

**Download icons in popular formats and sizes.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

**Comment by Anshuman-Verma** [comment]
Still multiple commits  :(

**Comment by aviaryan** [comment]
Iconfinder is a search engine type of tool. 
I think we should create a separate category for those things (called Search Engines). I remember there is a podcast finder and a RSS finder too. 
What do you think?

**Comment by Anshuman-Verma** [comment]
Yes search engine looks more suitable to me too,  just that it dealt with pictures so i kept it in graphics section. 
Should i switch it to search engine ?

**Comment by aviaryan** [comment]
> Still multiple commits :(

You may have created this new branch from unfurlr's branch, that's why. in GitHub UI, switch to master and then create a new branch to avoid this.

> Should i switch it to search engine ?

Yes, please.

**Comment by Anshuman-Verma** [comment]
Let's create new section for search engines and put these:
- RSS feed search
- Podcast feed extractor
- Iconfinder
Thoughts  ?

File: README.md [hunk 1]
```diff
@@ -214,6 +215,14 @@ See [plotting graphs examples](https://www.wolframalpha.com/examples/PlottingAnd
 * [Word Safety](http://wordsafety.com/) - Check if words are abusive in any languages.
 
 
+### Search Engines
+
+* [Iconfinder](http://www.iconfinder.com/) - Download all your favourite icons in one place.
+* [Emoji Search Online](http://www.webpagefx.com/tools/emoji-cheat-sheet/) - Intelligent emoji search engine.
+* [Podcast Feed Extractor](http://itunes.so-nik.com/) - Get RSS feed for podcasts.
+* [RSS Feed Search](http://ctrlq.org/rss/) - Find RSS feeds from search terms.
```
Comment (aviaryan) [inline README.md]: You need to remove duplicate Emoji Search, Podcast... etc from other sections.

**Comment by aviaryan** [comment]
@Anshuman-Verma Section looks perfect. Please remove the duplicate entries now.

File: README.md (hunk 1 above)
Comment (Anshuman-Verma) [inline README.md]: fixed

**Comment by Anshuman-Verma** [comment]
Done

**Pull request merged**